README.md

```diff
@@ -81,6 +81,49 @@ microovn.ovn-sbctl show
 microovn.ovn-appctl -t /var/snap/microovn/common/run/central/ovnnb_db.ctl cluster/status OVN_Northbound
 ```
 
+## Downscaling
+
+Removing MicroOVN cluster member is as easy as running
+
+```shell
+microovn cluster remove <member_name>
+```
+
+MicroOVN will attempt to gracefully remove services running on the removed
+member from the OVN cluster. You can watch logs on departing member for any
+indications of removal failure with:
+
+```shell
+snap logs -f microovn.daemon
+```
+
+If there are any issues during the removal process, you will have to take
```

Comment:
Do we have known issues for the removal process, if so should it be listed? If not, do we need to mention this?

Reply:
Of the top off my head I can think of:
* Service already stopped on the node that we are removing
* Network connectivity issues between OVN services
* Someone removed certificates on the node that we are removing
* Someone messed up configuration of ovn/microovn on the node that we are removing

I think there may be many problems, often outside of MicroOVN control. That's why I opted for vague wording here, and pointed user to the logs which will tell them what exactly failed during removal process.